hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/OutboxImpl.java

```diff
@@ -56,6 +57,7 @@
     private Object unfinishedSnapshotKey;
     private Object unfinishedSnapshotValue;
     private final AtomicLong lastForwardedWm = new AtomicLong(Long.MIN_VALUE);
+    private final AtomicLong snapshotSize = new AtomicLong(0);
```

Comment:
this metric doesn't make much sense to me. It seems to be always incremented, and not reset between snapshots, so how would it be useful?

Reply:
My thinking at the time was that it would give an estimate about the total amount of snapshot data written out by a `Processor` during the lifetime of a job. But now I realise that this is not data which accumulates, but data that continuously overwrites itself (ie. same keys all the time probably), so indeed, not useful...

Reply:
Have removed these changes.

Reply:
Hmm, number of snapshots might be interesting, even though one can estimate that from the interval..